src/lib/color-picker/color-picker-selector.component.html

```diff
@@ -1,18 +1,18 @@
 <div class="mcc-color-picker-selector" [ngStyle]="{ 'height.px': selectorHeight }">
-    <div #block class="mcc-picker-selector"></div>
-    <canvas #blockCanvas [height]="selectorHeight" width="240" id="picker"></canvas>
+    <div #block class="mcc-picker-selector" [style.height.px]="selectorHeight" [style.width.px]="selectorWidth"></div>
+    <canvas #blockCanvas [height]="selectorHeight" [width]="selectorWidth" id="picker"></canvas>
     <div #blockPointer class="mcc-picker-position" style="top: 0px;left: 220px;"></div>
 
-    <div #stripContainer class="mcc-colors-position" [ngClass]="{'alpha-enabled': showAlphaSelector}"
+    <div #stripContainer class="mcc-colors-position" [style.height.px]="selectorHeight" [ngClass]="{'alpha-enabled': showAlphaSelector}"
          style="background-position-y: 0px;">
-        <canvas #strip [height]="stripHeight" width="20" id="colors"></canvas>
+        <canvas #strip [height]="stripHeight" [width]="stripWidth" id="colors"></canvas>
+    </div>
+
+
+    <div *ngIf="showAlphaSelector" #alphaContainer class="mcc-alpha-position" [style.height.px]="selectorHeight" style="background-position-y: 0px;">
+        <canvas #alpha [height]="stripHeight" [width]="stripWidth" id="alpha"></canvas>
```

Comment:
These changes are causing some lag to select the color.

Reply:
How would you handle the sizes? Because with hard-coded Heights and widths it would not work if one changes the heigth via component-input. I did not experience any lag.

Reply:
I have tested again and the lag disappears.
Since the image used on strip selectors have a fixed width, I don't think it's a good idea to let users resize the width of these elements. 
Maybe just the color-picker?

You have to create these inputs on ```color-picker.component.ts``` too.

Reply:
on `color-picker.component.ts` i did not change the input. So there is still only the height. I will remove the @Input() frim the widths. I also changed the imaage on background of of the sliders to a very small image repeated as pattern x and y. So heigth change should be no problem.

Reply:
I pushed the changes